tests/test-cut-get-vars.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright(c) 2017 Michael Heerklotz
```

Comment:
2017 Should be for the FSF only (remove the first (C) line and set the year to 2017 for FSF).

Explanation:
I transferred my copyrights in 2015 to the FSF, so there are files with (C) Tim Rühsen + (C) FSF - the copyrigth year for the FSF should be >= 2015 and my copyright year should be <= 2015.